initializr-service/application.yml

```diff
@@ -456,6 +462,25 @@ initializr:
           versionRange: 1.3.0.M4
           groupId: org.springframework.cloud
           artifactId: spring-cloud-cluster-hazelcast
+    - name: Spring Cloud Services for Pivotal Cloud Foundry
+      bom: scs-bom
+      versionRange: "1.0.1.RELEASE"
```

Comment:
Is it not necessary to include the `versionRange`? I thought that was directing initializr to restrict this group of starters to the `1.0.1.RELEASE` version of the BOM.